protobuf/v1/replica.proto

```diff
@@ -130,3 +132,18 @@ message ListReplicaSnapshotsResponse {
 message DeleteReplicaSnapshotRequest {
   string snapshot_uuid = 1; // snapshot to be deleted
 }
+
+/// Request Parameters for Creating Snapshot Clone.
+message CreateSnapshotCloneRequest {
+  string snapshot_uuid = 1; // uuid of the snapshot
+  string clone_name = 2; // snapshot clone name
+}
+
+/// Request Parameters for List Snapshot Clone.
+message ListSnapshotCloneRequest {
+  optional string snapshot_uuid = 1;  // uuid of the snapshot
```

Comment:
optional snapshot clone uuid as well?

Reply:
If We will be writting the new SPDK API, then clone name can be save as lvol name. it wil not be a xattr. (Even the existing SPDK, consider it Lvol Name)
We need to have one xattr for Clone which will save `snapshot_uuid`. This will solve 2 purpose.
1. Identification of Lvol if it is Clone or not
2. Clone Lvol Creation Source.

I think it is logical to keep snapshot_uuid mandatory field for ListSnapshotClone. Else the result will be big to return.